independent-publisher-connectors/Mensagia/apiProperties.json

```diff
@@ -0,0 +1,25 @@
+{
+    "properties": {
+        "connectionParameters": {
+            "api_key": {
+                "type": "securestring",
+                "uiDefinition": {
+                    "displayName": "Api Key",
+                    "description": "Api Key for this API.",
+                    "tooltip": "Set your Api Key.",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#f8f8f8",
```

Comment:
@AfriSR You can not set a different iconBrandColor for an independent publisher connector. Please change it to `"da3b01"`.

A [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json) for apiProperties.json